src/app/app.openWeatherMapsService.js

```diff
@@ -0,0 +1,19 @@
+export default class OpenWeatherMapService {
+    constructor($http) {
+        this.$http = $http;
+    }
+    getCurrentWeather(city) {
+        const apiKey = '608c0e7fd9bce774dd414deb10de6e52';
```

Comment:
You must not commit your secrets into a public repository.

See https://github.com/jLagosCarrera/MeteoApp/pull/4#discussion_r280689452 

You can ask Julio about how to solve this